docs/pages/project/contributing/contributing-docs.md

```diff
@@ -485,6 +485,32 @@ You can include block quotes to emphasize text.
 
 ## Development
 
+### Adding Integration Specific Information to Individual Integration Pages
+
+Integration pages ([example](https://docs.meshery.io/extensibility/integrations/aws)) are automatically generated, however, integration specific documentation is often needed. 
+
+[modelscustominfo](https://github.com/meshery/meshery/tree/master/docs/_modelscustominfo) collection holds custom markdown files. Follow these steps:
```

Comment:
should this be `https://github.com/meshery/meshery/tree/master/docs/_models` instead